src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
```

Comment:
I suggest to use 
```
execute_tool_on_source_files(project, "pylint", command_and_arguments, include_test_sources=?)
```
We will get two advantages:
1) provide possibility to include test_sources with property `pylint_include_test_sources`
2) remove possible problems with pylint: `execute_tool_on_modules` uses `discover_modules` which returns list of all packages AND modules. For example, I have package `pckg` with modules `a`, `b` and `c`. `discover_modules` returns list: `[ 'pckg', 'pckg.a', 'pckg.b', 'pckg.c']` and pylint report problem with duplicated code into files (it will parse `pckg.a` twice).